hastory-server-data/src/Hastory/Server/Data.hs

```diff
@@ -27,17 +27,17 @@ share
   [persistLowerCase|
 
 User
-    name Username sql=citext
+    name Username sqltype="VARCHAR COLLATE NOCASE"
```

Comment:
I had no idea that this was possible :p
Awesome.

Reply:
:-)